integration_test/third_party_apps_data/applications/aerospike/metadata.yaml

```diff
@@ -86,6 +86,13 @@ expected_metrics:
     monitored_resource: gce_instance
     type: workload.googleapis.com/aerospike.node.memory.free
     value_type: INT64
+expected_logs:
+  - log_name: syslog
+    fields:
+      - name: jsonPayload.message
+        value_regex: 'asd(\[[0-9]+\])*:'
```

Comment:
Should this have an anchor (e.g., `^asd(\[[0-9]+\])*:`)?

Reply:
There is timestamp before `asd`, so it's not the starting point of the message

Reply:
Do we at least need a word boundary? Otherwise it could match strings like "wasd[3]" from some random unrelated process…

Reply:
That makes sense. Added a space.